pkg/cmd/controller/controller_build.go

```diff
@@ -789,30 +789,38 @@ func updateForStage(si *tekton.StageInfo, a *v1.PipelineActivity) {
 	containersTerminated := false
 
 	if si.Pod != nil {
+		var stageSteps []v1.CoreActivityStep
 		pod := si.Pod
 		_, containerStatuses, _ := kube.GetContainersWithStatusAndIsInit(pod)
 		containersTerminated = len(containerStatuses) > 0
-		for _, c := range containerStatuses {
-			name := strings.Replace(strings.TrimPrefix(c.Name, "build-step-"), "-", " ", -1)
-			name = strings.Replace(strings.TrimPrefix(c.Name, "step-"), "-", " ", -1)
-			title := strings.Title(name)
+		for i, c := range containerStatuses {
```

Comment:
Personally I am not a big fan of single character variables except for indeces. _container.Name_ is imo so much easier to make sense of than c.Name.
Just saying.